Game/ObjectsCreation.pde

```diff
@@ -2,4 +2,8 @@ void createGameObjects() {
 
   Furniture a = furnitureFactory.createWall(4, 3);
   board.put_on_board(a, a.p, a.r);
+  Furniture b = furnitureFactory.createWall(4, 4);
+  board.put_on_board(b, b.p, b.r);
+  Furniture c = furnitureFactory.createWall(5, 4);
+  board.put_on_board(c, c.p, c.r);
 }
```

Comment:
As you can see this doesn´t scale to well. I suggest two different options:

1. we add another private method inside the file "createGameObjects" that calls "board.put_on_board" for us and we only have to pass the Furniture-object.

2. OR we change the method "board.put_on_board" to only need 1 parameter. The Furniture, since it already has the other information saved within